src/caselawclient/__init__.py

```diff
@@ -15,7 +29,7 @@
 or in your projects `requirements.txt` with:
 
 ```text
-ds-caselaw-marklogic-api-client~=12.0.0
+ds-caselaw-marklogic-api-client{__version_string__}
```

Comment:
feels weird that the version string has the `~=` prefix, but not a problem

Reply:
A quicker fix would have been just removing the version number entirely?
And we should probably move to poetry long term.

Reply:
We should, but this is included for those who haven't. I've fixed up how we define this.